### PR TITLE
Update success stories (id)

### PR DESCRIPTION
**Pull request by meisyal**
Changes:

* Remove dead links
* Update content to  ensure the content is the same as English page

@gozali I need your help to review this. Thanks.

### CODE REVIEW

File: id/documentation/success-stories/index.md [hunk 1]
```diff
@@ -9,85 +9,85 @@ yang menggunakannya sebagai hobi. Di sini Anda akan menemukan sebagian
...
   yang tersebar di seluruh dunia.
 
+#### Aplikasi Web
+
+* [Basecamp][9], sebuah aplikasi manajemen proyek berbasis *web* yang
+  dikembangkan oleh [37signals][10], diprogram sepenuhnya dalam bahasa Ruby.
 
+* [A List Apart][11], sebuah majalah digital bagi para *webmaster* yang telah
+  berdiri sejak tahun 1997, telah direnovasi dan menggunakan aplikasi
+  khusus yang dikembangkan menggunakan Ruby on Rails.
```
Comment (gozali) [inline id/documentation/success-stories/index.md]: maybe we can put Github and Shopify here?

Comment (meisyal) [reply id/documentation/success-stories/index.md]: Hmm 🤔

The English version of success stories doesn't have that. Shouldn't we follow that version?

Comment (JuanitoFatas) [reply id/documentation/success-stories/index.md]: If would like to do so, please change upstream (English) first.

Comment (gozali) [reply id/documentation/success-stories/index.md]: I will do the changes on upstream later, I think we can merge this PR first?

**Comment by meisyal** [comment]
As @gozali said, he will do the changes on upstream later. So, this PR can be merged @stomar but it has a conflict.

**Comment by meisyal** [comment]
I just resolved the conflict and merged this PR.

**Pull request merged**